drivers/CoreVideo/tests/CoreLCDDriverOTM8009A_test.cpp

```diff
@@ -45,7 +45,13 @@ TEST_F(CoreOTM8009ATest, initialize)
 
 TEST_F(CoreOTM8009ATest, setLandscapeOrientation)
 {
-	auto expected_instruction_array = ElementsAre(lcd::otm8009a::madctr::command, 0xA0);
+	auto _settings				 = uint8_t {0x00};
+	auto set_landscape_mode		 = [&] { _settings |= (1 << 5); };
+	auto set_horizontal_symmetry = [&] { _settings |= (1 << 6); };
+	set_landscape_mode();
+	set_horizontal_symmetry();
```

Comment:
si tu veux éviter d'appeler les lambda, tu peux faire ça pour appeler direct:

```suggestion
	auto set_landscape_mode		 = [&] { _settings |= (1 << 5); }();
	auto set_horizontal_symmetry = [&] { _settings |= (1 << 6); }();
```

Reply:
Mh, ça génère une erreur comme quoi la variable est de type `void`